go-controller/pkg/cni/helper_linux.go

```diff
@@ -252,6 +253,21 @@ func setupInterface(netns ns.NetNS, containerID, ifName string, ifInfo *PodInter
 			ifnameSuffix = fmt.Sprintf("_%d", containerVeth.Index)
 		}
 
+		if gwLLA != "" {
```

Comment:
This field won't be set for non-vms right? that's how you guarantee this has no effect for normal pods?

Reply:
A layer2 pod would want to use RAs to configure itself, on that case this still applies